api/src/pcapi/core/users/models.py

```diff
@@ -657,6 +660,21 @@ def has_partner_page(self) -> bool:
         return has_partner_page
 
 
+class DiscordUser(PcObject, Base, Model):
+    __tablename__ = "discord_user"
+
+    userId = sa.Column(sa.BigInteger, sa.ForeignKey("user.id", ondelete="CASCADE"), primary_key=True)
+    user: orm.Mapped["User"] = orm.relationship("User", back_populates="discordUser")
+    discordId = sa.Column(sa.Text, nullable=True, unique=True)
+    has_access = sa.Column(sa.Boolean, nullable=False, server_default=expression.true())
+    is_banned = sa.Column(sa.Boolean, nullable=False, server_default=expression.false(), default=False)
+    last_updated = sa.Column(sa.DateTime, nullable=False, server_default=sa.func.now(), onupdate=sa.func.now())
```

Comment:
Pour l'homogénéité, les noms de champs des modèles SQLAlchemy devraient être en camelCase.